TypeCobol/Compiler/CodeModel/SymbolTable.cs

```diff
@@ -344,28 +344,44 @@ void SearchChildren(IReadOnlyList<Node> children)
             }
         }
 
-        public DataDefinition GetRedefinedVariable(DataRedefines redefinesNode, SymbolReference symbolReference)
+        public DataDefinition GetRedefinedVariable(DataRedefines redefinesNode)
         {
+            var redefinesDataName = redefinesNode.CodeElement.RedefinesDataName.Name;
             var childrens = redefinesNode.Parent.Children;
             int index = redefinesNode.Parent.IndexOf(redefinesNode);
 
             while (index >= 0)
             {
-                CommonDataDescriptionAndDataRedefines child = childrens[index].CodeElement as CommonDataDescriptionAndDataRedefines;
+                var child = childrens[index];
+                if (child == null)
+                {
+                    return null;
+                }
```

Comment:
Not needed. I mean, could the `Children` collection contain null values ? It may contain items which are not data but `null` ??

Reply:
Check removed.